Problems with pdflatex in an Italian document built with sphinx

In an earlier case I had problems with shorthands in a Turkish document built with sphinx in pdflatex: [https://github.com/sphinx-doc/sphinx/issues/2397](https://github.com/sphinx-doc/sphinx/issues/2397).

Now I have a similar issue with an Italian document in the same system:

`[exec] C:/foo/bar.tex:141:
BUILD FAILED`

Line 141 of bar.tex reads `\shorthandoff{"}`.

I was able to fix this in a way similar to the earlier case: I removed Italian from get_shorthandoff function from sphinx\writers\latex.py. I.e. I modified this 

```
def get_shorthandoff(self):
      shortlang = self.language.split('_')[0]
       if shortlang in ('de', 'ngerman', 'sl', 'slovene', 'pt', 'portuges',
             'es', 'spanish', 'nl', 'dutch', 'pl', 'polish', 'it',
                    'italian'):
             return '\\shorthandoff{"}'
       elif shortlang in ('tr', 'turkish'):
             return '\\shorthandoff{=}'
       return ''
```

to this

```
def get_shorthandoff(self):
      shortlang = self.language.split('_')[0]
       if shortlang in ('de', 'ngerman', 'sl', 'slovene', 'pt', 'portuges',
             'es', 'spanish', 'nl', 'dutch', 'pl', 'polish'):
             return '\\shorthandoff{"}'
       elif shortlang in ('tr', 'turkish'):
             return '\\shorthandoff{=}'
       return ''
```

Tested with sphinx 1.4.5. Just to make myself clear: This is not a new issue with a new version of sphinx. I have a new document (new language) with which this problem appeared.

--- a/sphinx/writers/latex.py
+++ b/sphinx/writers/latex.py
@@ -93,7 +93,7 @@ class ExtBabel(Babel):
         if shortlang in ('de', 'ngerman', 'sl', 'slovene', 'pt', 'portuges',
                          'es', 'spanish', 'nl', 'dutch', 'pl', 'polish', 'it',
                          'italian'):
-            return '\\shorthandoff{"}'
+            return '\\if\\catcode`\\"\\active\\shorthandoff{"}\\fi'
         elif shortlang in ('tr', 'turkish'):
             return '\\shorthandoff{=}'
         return ''

